### PR TITLE
document data set in the R folder

### PR DESCRIPTION
**Pull request by brennanpincardiff**
Hi, I think documenting the data set is good practice. 
I hope it is OK but I would invite you to check it, please, Dom. 
Best wishes, 
Paul

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#'     \item{first}{given or first name}
+#'     \item{second}{family or surname}
+#'     \item{role}{organiser, speaker or delegate}
+#'     \item{email}{contact details}
+#'     \item{affiliation}{organisation}
+#'     \item{include_gender_pronouns}{YES or NO}
+#'     \item{gender_pronouns}{can be a variety}
+
+#' }
+#' @return A data frame with 320 rows and 9 variables
```
Comment (dokato) [inline R/data.R]: there's 6 rows only, right?

**Review by dokato — COMMENTED** [review]
Thanks. PTAL at the minor comment, once changed ready to merge

File: R/data.R (hunk 1 above)
Comment (brennanpincardiff) [reply R/data.R]: Correct. Changed.

**Comment by brennanpincardiff** [comment]
I've added a first test now too. 
Can you add the hacktober label, please?
Checked with Test package and passes :-)

**Comment by dokato** [comment]
Great, LGTM

**Pull request merged**